dicom_utils/cli/dicom2img.py

```diff
@@ -26,10 +28,50 @@ def get_parser(parser: ArgumentParser = ArgumentParser()) -> ArgumentParser:
     return parser
 
 
+def to_collage(images: List[ndarray]) -> ndarray:
+    num_images = len(images)
+
+    assert num_images != 0, "There must be at least one image."
+    assert all(len(i.shape) == 3 for i in images), "The images must have 3 channels."
```

Comment:
Should this assertion message be reworded as "3 dimensions"? 3 channels implies RGB

Reply:
Yes, good catch